### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by Luis-Hebendanz**
I documented the build process and added `sse` to the feature list and a little debug help section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -87,13 +157,28 @@ dd if=target/x86_64-bootloader/release/bootloader.bin of=/dev/sdX && sync
 
 Where sdX is the device name of your USB stick. **Be careful** to choose the correct device name, because everything on that device is overwritten.
 
+## Debugging
+Set a breakpoint at address `0x7c00`. Disassemble instructions with gdb:
```
Comment (bjorn3) [inline README.md:161]: This address will differ between versions of bootloader and possibly between assembler versions.

Comment (rybot666) [reply README.md:161]: That's the address BIOS jumps to to start code execution, we'll always have *something* there

Comment (bjorn3) [reply README.md:161]: Ah, ok.

**Comment by ethindp** [comment]
I disagree; when the bootloader gains UEFI support 0x7C00 will no longer be a constant address, I believe.

**Comment by Luis-Hebendanz** [comment]
> I disagree; when the bootloader gains UEFI support 0x7C00 will no longer be a constant address, I believe.

I mean, the documentation documents the current behavior if this changes the documentation should be changed accordingly

**Review by phil-opp — APPROVED** [review]
Looks good to me, thanks a lot!

I agree that we can leave the `0x7c00` debug address in for now, we will have to update the documentation in many places for the new UEFI bootloader anyway.

**Pull request merged**